Sources/OpenCombine/Publishers/Publishers.Map.swift

```diff
@@ -10,20 +10,31 @@ extension Publisher {
     /// Transforms all elements from the upstream publisher with a provided closure.
     ///
     /// - Parameter transform: A closure that takes one element as its parameter and returns a new element.
-    /// - Returns: A publisher that uses the provided closure to map elements from the upstream publisher to new elements that it then publishes.
```

Comment:
Hey, great work :)
Why these reformatting changes were made? As I can see code fits swiftlint rules even without these changes. Many unnecessary changes which have no relation to feature (tryMap) complicate code review process. Could u please revert these changes?

Reply:
I've accidentally configured SwiftLint in such a way that comments were allowed to ignore the line length limit. Fixed that in master,